scripts-available/CDB_Groups.sql

```diff
@@ -0,0 +1,138 @@
+-- Creates a new group
+CREATE OR REPLACE
+FUNCTION cartodb.CDB_Group_CreateGroup(group_name text)
+    RETURNS VOID AS $$
+DECLARE
+    cdb_group_role TEXT;
+BEGIN
+    cdb_group_role := cartodb._CDB_Group_GroupRole(group_name);
+    IF NOT EXISTS ( SELECT 1 FROM pg_roles WHERE rolname = cdb_group_role )
+    THEN
+        EXECUTE 'CREATE ROLE "' || cdb_group_role || '" NOLOGIN;';
```

Comment:
I mixed two roles here:
- Organization administrator role is created on extension install, just like organization role is.
- Group creation checked role existence, which is not needed: PostgreSQL will throw an exception if that happens. I've removed the check.